firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -177,6 +183,7 @@ private void setCommonRequestHeaders(HttpURLConnection httpURLConnection) {
     // Headers to denote that the request body is a JSONObject.
     httpURLConnection.setRequestProperty("Content-Type", "application/json");
     httpURLConnection.setRequestProperty("Accept", "application/json");
+    httpURLConnection.setConnectTimeout(31000);
```

Comment:
We shouldn't set the connection timeout on the client since that'll prevent us from extending it on the backend.

Reply:
Removed